data/json/vehicle_parts.json

```diff
@@ -5104,5 +5104,82 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "count": [1, 2]}
         ]
+    },{
+        "type" : "vehicle_part",
+        "id" : "plow",
+        "name" : "plow",
+        "symbol" : "&",
+        "color" : "blue",
+        "broken_symbol" : "*",
+        "broken_color" : "light_blue",
+        "damage_modifier" : 40,
+        "durability" : 200,
+        "item" : "v_plow_item",
+        "difficulty" : 4,
+        "location" : "under",
+        "bonus": 1500,
+        "power" : -100,
+        "flags" : ["PLOW"],
+        "//":"Bonus for a plow part determines the friction added in k_friction while active",
+        "breaks_into" : [
+            {"item": "plastic_chunk", "count": [1, 2]}
+        ]
+    },{
+        "type" : "vehicle_part",
```

Comment:
Seed drills and reapers are also going to be a drag on a vehicle.  If I understand correctly, they are either going to be coupled to the vehicle powertrain directly, or they will get power from being drug across the ground (think of the kind of children's toy that is actuated by turning its wheels).  Either way, they'll be a drain on power just like plows, perhaps with different values.